CMakeLists.txt

```diff
@@ -56,6 +57,16 @@ check_library_exists(m pow "" HAVE_LIBM)
 check_include_file("dirent.h" HAVE_DIRENT_H)
 check_symbol_exists(strcasecmp "strings.h" HAVE_DECL_STRCASECMP)
 check_symbol_exists(strncasecmp "strings.h" HAVE_DECL_STRNCASECMP)
+check_c_source_compiles("
```

Comment:
`_div64` is instrinsic and doesn't work with `check_symbol_exists`: [docs](https://cmake.org/cmake/help/latest/module/CheckSymbolExists.html) I assume this is because instrinsic is not a regular function and is always inlined.